flowforge/materials/__init__.py

```diff
@@ -0,0 +1,12 @@
+from .Solid import Graphite, SS316H
+from .Fluid import FLiBe_UF4, Hitec
+
+material_database = {
```

Comment:
I'm not sure how I feel about combining these

Reply:
also, can you add to this outside of FlowForge.  This would be beneficial for protected data.

Reply:
I can double check that, but after talking with Aaron about it, he agrees that you can still add materials to the database after importing the object.

Reply:
can you confirm this?  Lets seperate the lists

Reply:
```python
from flowforge.materials import solid_material_database
from flowforge.materials.Solid import User_Solid

# Define the material properties for the aluminum cladding (estimated properties)
aluminum = User_Solid(
    name="aluminum",
    k_funct=lambda T: 237.0,
    dens_funct= lambda T: 2700.0,
    cp_funct=lambda T: 900.0
)

# Output 1
print(solid_material_database)

# Output 2
solid_material_database["aluminum"] = aluminum
print(solid_material_database)
```

Output 1:
```
{'graphite': <class 'flowforge.materials.Solid.Graphite'>, 
'ss316h': <class 'flowforge.materials.Solid.SS316H'>}
```

Output 2:
```
{'graphite': <class 'flowforge.materials.Solid.Graphite'>, 
'ss316h': <class 'flowforge.materials.Solid.SS316H'>, 
'aluminum': <flowforge.materials.Solid.User_Solid object at 0x7f354ea91790>}
```

Reply:
The thing I'm just noticing now is that the default materials are uninitialized, but the one that I added is initialized already. So should I go ahead and initialize the other materials on FlowForge's database?

Reply:
This holds true even from file to file:

`debugging.py`:
```python
from flowforge.materials import solid_material_database
from flowforge.materials.Solid import User_Solid

def add_materials():
    # Define the material properties for the aluminum cladding
    aluminum = User_Solid(
        name="aluminum",
        k_funct=lambda T: 237.0,
        dens_funct=lambda T: 2700.0,
        cp_funct=lambda T: 900.0
    )

    solid_material_database["aluminum"] = aluminum
```

`fake_driver.py`:
```python
from debugging import add_materials
from flowforge.materials import solid_material_database

add_materials()

print(solid_material_database)
```

output:
```
{'graphite': <class 'flowforge.materials.Solid.Graphite'>, 
'ss316h': <class 'flowforge.materials.Solid.SS316H'>, 
'aluminum': <flowforge.materials.Solid.User_Solid object at 0x7f33ff6ea310>}
```